### PR TITLE
Update vignettes.Rmd

### PR DESCRIPTION
**Pull request by MikeJohnPage**
Remove expired website.

### CODE REVIEW

File: vignettes.Rmd [hunk 1]
```diff
@@ -183,7 +183,7 @@ Here are some resources we've found helpful:
 
 -   Literally anything written by Kathy Sierra.
     She is not actively writing at the moment, but her content is mostly timeless and is full of advice about programming, teaching, and how to create valuable tools.
-    See her original blog, [Creating passionate users](https://headrush.typepad.com/), or the site that came after, [Serious Pony](https://seriouspony.com/blog/).
```
Comment (jennybc) [inline vignettes.Rmd:186]: What do you think about using the Wayback Machine instead?

https://web.archive.org/web/20230411010913/https://seriouspony.com/blog/

Comment (MikeJohnPage) [reply vignettes.Rmd:186]: Nice idea. I've added another commit with the archived link in case you want to go with that.

**Comment by jennybc** [comment]
Thanks!

**Comment by MikeJohnPage** [comment]
Thanks to you and Hadley for such a great book!

**Pull request merged**